docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3"
+services:
+  grafana:
+    image: grafana/grafana-enterprise
+    container_name: grafana
+    ports:
+      - 3000:3000
+    environment:
+      - GF_ALLOW_LOADING_UNSIGNED_PLUGINS=lightstep-metrics-datasource
+      - GF_LOG_LEVEL=debug
+      - GF_DEFAULT_APP_MODE=development
+    volumes:
+      - grafana-data-lmd:/var/lib/grafana
```

Comment:
Does this need to match what's in the README? `grafana-data` versus `grafana-data-lmd`?

Reply:
yes, just updated :)